storybook/pages/AssetsViewPage.qml

```diff
@@ -99,6 +102,21 @@ SplitView {
             Layout.fillWidth: true
             areAssetsLoading: loadingCheckbox.checked
             assets: d.walletAssetStore.groupedAccountAssetsModel
+            controller: ManageTokensController {
```

Comment:
Just an implementation detail (will not have any impact)... Would it make sense to do it other way around, creating it in the store, and call the store here?

Reply:
That was my (alternative) idea as well but this setup would make it even more complicated for StoryBook